pkg/apis/operator/v1alpha1/types.go

```diff
@@ -163,6 +166,16 @@ type Maintenance struct {
 	TimeWindow gardencorev1beta1.MaintenanceTimeWindow `json:"timeWindow"`
 }
 
+// ControlPlane holds information about the general settings for the control plane of the virtual garden cluster.
+type ControlPlane struct {
+	// HighAvailability holds the configuration settings for high availability settings.
+	// +optional
+	HighAvailability *HighAvailability `json:"highAvailability,omitempty"`
```

Comment:
Can we make this immutable if set once, [ref](https://kubernetes.io/blog/2022/09/29/enforce-immutability-using-cel/#immutablility-after-first-modification)?